docs/src/_includes/components/rule-categories.macro.html

```diff
@@ -32,7 +32,8 @@
     <div class="rule-category">
         <span class="rule-category__icon">❄️ <span class="visually-hidden">Frozen</span></span>
         <p class="rule-category__description">
-            This rule is currently frozen and is not accepting changes.
+            This rule is currently frozen and is not accepting <a
+                href="https://github.com/eslint/eslint/#frozen-rule-updates">feature requests</a>
```

Comment:
> feature requests

[Accessibility] From [Understanding SC 2.4.4 Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html): links need to have text that makes sense on their own. This one reads as the text _"feature requests"_ which isn't a very helpful description or title for the page it links to.

Two common ways of fixing this are to either:

* Add a `title` attribute to the link: `<a href="..." title="ESLint Frozen rule updates">feature requests</a>
* Include more text in the link: `<a href="...">not accepting feature requests</a>`

I'd personally use the `title` approach so as to not change what the link visually looks like.